github3/repos/repo.py

```diff
@@ -2447,7 +2455,7 @@ def _update_attributes(self, repo):
         self.has_wiki = repo['has_wiki']
         self.homepage = repo['homepage']
         self.language = repo['language']
-        self.original_license = repo['license']
+        self.original_license = repo.get('license', None)
```

Comment:
`repo.get()` returns `None` implicitly. Can you share why you specified `None` explicitly here?

Reply:
purely to match the code style at https://github.com/sigmavirus24/github3.py/blob/aa50e04ebf02f8d9498f3f6024dd30d9000b8361/github3/repos/repo.py#L2458-L2460 and https://github.com/sigmavirus24/github3.py/blob/aa50e04ebf02f8d9498f3f6024dd30d9000b8361/github3/repos/repo.py#L2463-L2465

Have no attachment to doing it that way, should I update all 3 calls to use the same approach? Or just this new occurrence?

Reply:
Might as well update all of them. I don't know what I was thinking. :laughing: